### PR TITLE
feat(ui5-badge): new property added

### PR DESCRIPTION
**Pull request by GerganaKremenska**

### CODE REVIEW

File: packages/main/src/types/BadgeSize.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Defines badge size types.
```
Comment (BorisDafov) [inline packages/main/src/types/BadgeSize.ts:2]: "size types" is not the best wording. Maybe:
Defines badge size options.
or
Predefined sizes for the badge.

File: packages/main/src/types/BadgeSize.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * @public
+ */
+enum BadgeSize {
+	/**
+	 * Standard size of the badge
```
Comment (BorisDafov) [inline packages/main/src/types/BadgeSize.ts:7]: Pls. double check. "Standard" is OK, but implies that it should always be the default value.

**Review by BorisDafov — CHANGES_REQUESTED** [review]

File: packages/main/src/types/BadgeSize.ts (hunk 1 above)
Comment (GerganaKremenska) [reply packages/main/src/types/BadgeSize.ts:2]: "Predefined sizes for the badge". sounds good to me

File: packages/main/src/types/BadgeSize.ts (hunk 2 above)
Comment (GerganaKremenska) [reply packages/main/src/types/BadgeSize.ts:7]: "S" is for small, this is old wording.

**Comment by BorisDafov** [comment]
Form UA perspective I approve the PR...

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**